src/Algebra/Morphism/Construct/Identity.agda

```diff
@@ -247,3 +248,28 @@ module _ (L : RawLoop c ℓ) (open RawLoop L) (refl : Reflexive _≈_) where
     { isLoopMonomorphism = isLoopMonomorphism
     ; surjective = _, refl
     }
+
+------------------------------------------------------------------------
+-- KleeneAlgebra
+
+module _ (K : RawKleeneAlgebra c ℓ) (open RawKleeneAlgebra K) (refl : Reflexive _≈_) where
+  open KleeneAlgebraMorphisms K K
+
+  isKleeneAlgebraHomomorphism : IsKleeneAlgebraHomomorphism id
+  isKleeneAlgebraHomomorphism = record
+    { isSemiringHomomorphism = isSemiringHomomorphism _ refl
+    ; ⋆-homo = λ _ → refl
+    }
+
+  isKleeneAlgebraMonomorphism : IsKleeneAlgebraMonomorphism id
+  isKleeneAlgebraMonomorphism = record
+    { isKleeneAlgebraHomomorphism = isKleeneAlgebraHomomorphism
+    ; injective = id
+    }
+
+  isKleeneAlgebraIsomorphism : IsKleeneAlgebraIsomorphism id
+  isKleeneAlgebraIsomorphism = record
+    { isKleeneAlgebraMonomorphism = isKleeneAlgebraMonomorphism
+    ; surjective = _, refl
```

Comment:
This now fails: but `id` provides the right proof in place of `refl`.
Diagnosis/solution: again, changes to `Function` hierarchy suggest rebasing this PR against `master` and fixing this error.